test/Bread.t.sol

```diff
@@ -238,27 +238,279 @@ contract BreadTest is Test {
+            10536699826174859678,
+            20000000000000000000,
+            10000000000000000000,
+            15000000000000000000,
+            10500000000000000000,
+            49526000000000000000,
+            100000000000000000000,
+            999758363687661493842,
+            95000000000000000000,
+            10000000000000000000,
+            1300000000000000000,
+            20000000000000000000,
+            100000000000000000000,
+            377379625237371992,
+            10530515618992923096,
+            10000000000000000000,
+            15000000000000000000,
+            10991627315958077337,
+            145000000000000000000,
+            12431646314320140932,
+            1000000000000000000,
+            600000000000000000000,
+            90000000000000000000,
+            10000000000000000000,
+            50000000000000000000,
+            65000000000000000000,
+            25000000000000000000,
+            74991872265556673834,
+            2687001846091446,
+            5298983838010000,
+            10000000000000000000,
+            10000000000000000000,
+            15000000000000000000,
+            10000000000000000000,
+            20001000050000000000,
+            21000000000000000000,
+            50000000000000000000,
+            20000000000000000000,
+            17000000000000000000,
+            1005216355662533665,
+            1110069562590244408,
+            50000000000000000000,
+            50000000000000000000,
+            6000000000000000000,
+            40000000000000000000,
+            20000000000000000000,
+            2000000000000000000,
+            5560000000000000000,
+            6999999998499474000,
+            3000000001500526000,
+            20000000000000000000,
+            69078875220932980185,
+            10000000000000000000,
+            2176175970114874216,
+            9732031901839308852,
+            12416479916753708,
+            710455193108857,
+            15000000000000000000,
+            5000000000000000000,
+            14990000000000000000,
+            4561076106274262739985,
+            350000000000000000000,
+            500000000000000000000,
+            30000000000000000000,
+            500000000000000000,
+            99823105817665230801,
+            4000000000000000000,
+            50000000000000000000,
+            50500000000000000000,
+            4523967209611154265025,
+            331314182184129242087,
+            695590009403958563,
+            11000000000000000000,
+            136700000000000000000,
+            1,
+            9999000000000000000,
+            9800999999999999999,
+            1000000000000000000,
+            2238541110297335924,
+            500000000000000000,
+            23070145829119381495401
+        ];
+        breadToken.batchMint{value:polygontotalSupply}(migration_amounts, migration_addresses);
+        assertEq(breadToken.totalSupply(), polygontotalSupply + 1 ether);
```

Comment:
would be nice to throw in some balance checks here

Reply:
maybe with fuzzing, opened #19